standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -7420,6 +7420,64 @@ public boolean delete_partition_column_statistics(String dbName, String tableNam
     return ret;
   }
 
+  @Override
+  public boolean delete_partition_column_statistics_req(DeletePartitionColumnStatisticsRequest req) throws TException {
+    String dbName = req.getDb_name();
+    String tableName = req.getTbl_name();
+    List<String> colNames = req.getCol_names();
+    String partName = req.getPart_name();
+    String engine = req.getEngine();
+    dbName = dbName.toLowerCase();
+    String[] parsedDbName = parseDbName(dbName, conf);
+    tableName = tableName.toLowerCase();
+    if (colNames != null) {
+      for (String colName : colNames) {
+        colName = colName.toLowerCase();
+        startFunction("delete_column_statistics_by_partition",": table=" +
+                TableName.getQualified(parsedDbName[CAT_NAME], parsedDbName[DB_NAME], tableName) +
+                " partition=" + partName + " column=" + colName);
+      }
+    }
+    boolean ret = false, committed = false;
+
+    getMS().openTransaction();
+    try {
+      List<String> partVals = getPartValsFromName(getMS(), parsedDbName[CAT_NAME], parsedDbName[DB_NAME], tableName, partName);
```

Comment:
this `partName` in thrift defines as `optional`, should we take care of the nullable `partName`. To make the interface more generic, can we define the `partName` as `List<String>`, so we can delete multiple partitions' statistics?